### PR TITLE
Cleanup JS part as suggested under users/_account.html.erb

### PR DESCRIPTION
**Pull request by sibis**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

This PR contains the Javascript code cleanup as suggested by @rhymes & @lightalloy. 
 
## Related Tickets & Documents

Based on the discussions here,
https://github.com/thepracticaldev/dev.to/pull/4480#discussion_r343698671

### CODE REVIEW

File: app/views/users/_account.html.erb [hunk 1]
```diff
@@ -121,30 +121,17 @@
 </h4>
 <br>
 <script>
-  var deleteAccountBtn = document.getElementById('delete__account__btn')
-  var deleteAccountUsernameInput = document.getElementById('delete__account__username__field')
-  var deleteAcccountVerificationInput = document.getElementById('delete__account__verification__field')
+  var deleteAccountBtn = document.getElementById('delete__account__btn');
```
Comment (nickytonline) [inline app/views/users/_account.html.erb]: If you’re in here cleaning up, might as well convert `var`s to `const`s.

Comment (sibis) [reply app/views/users/_account.html.erb]: Have taken care of, @nickytonline. Thanks :)

**Comment by sibis** [comment]
Sorry closed by mistake. Re-opened it again. :)

**Comment by benhalpern** [comment]
I just want to double check....

We're confident that these changes will work across different browsers given that the code is running in the HTML and not being pre-processed?

**Comment by nickytonline** [comment]
If you're concerned about the usage of `const`, it is used elsewhere in non-babelified scripts already, e.g. https://github.com/thepracticaldev/dev.to/blob/master/app/assets/javascripts/initializers/initializeBaseUserData.js#L27

Also, even for webpack/babel, we support `> 1%` for browsers, so `const` should be fine since it is not transpiled to `var`.

https://github.com/thepracticaldev/dev.to/blob/master/.babelrc#L3

https://browserl.ist/?q=%3E+1%25

image

**Pull request merged**